linode_api4/linode_client.py

```diff
@@ -430,6 +429,120 @@ def ssh_key_upload(self, key, label):
         return ssh_key
 
 
+class LKEGroup(Group):
+    """
+    Encapsulates LKE-related methods of the :any:`LinodeClient`.  This
+    should not be instantiated on its own, but should instead be used through
+    an instance of :any:`LinodeClient`::
+
+       client = LinodeClient(token)
+       instances = client.lke.clusters() # use the LKEGroup
+
+    This group contains all features beneath the `/lke` group in the API v4.
+    """
+    def versions(self, *filters):
+        """
+        Returns a :any:`PaginatedList` of :any:`KubeVersion` objects that can be
+        used when creating an LKE Cluster.
+
+        :param filters: Any number of filters to apply to the query.
+
+        :returns: A Paginated List of kube versions that match the query.
+        :rtype: PaginatedList of KubeVersion
+        """
+        return self.client._get_and_filter(KubeVersion, *filters)
+
+    def clusters(self, *filters):
+        """
+        Returns a :any:`PaginagtedList` of :any:`LKECluster` objects that belong
+        to this account.
+
+        :param filters: Any number of filters to apply to the query.
+
+        :returns: A Paginated List of LKE clusters that match the query.
+        :rtype: PaginatedList of LKECluster
+        """
+        return self.client._get_and_filter(LKECluster, *filters)
+
+    def cluster_create(self, region, label, node_pools, **kwargs):
+        """
+        Creates an :any:`LKECluster` on this account in the given region, with
+        the given label, and with node pools as described.  For example::
+
+           client = LinodeClient(TOKEN)
+
+           # look up Region and Types to use.  In this example I'm just using
+           # the first ones returned.
+           target_region = client.regions().first()
+           node_type = client.linode.types()[0]
+           node_type_2 = client.linode.types()[1]
+
+           new_cluster = client.lke.cluster_create(
+               target_region,
+               "example-cluster",
+               [client.lke.node_pool(node_type, 3), client.lke.node_pool(node_type_2, 3)]
+            )
+
+        :param region: The Region to create this LKE Cluster in.
+        :type region: Region of str
```

Comment:
NB: just "str" for the value, or better way to describe it?

Reply:
`str` is the name of the [python string type](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str), so I think this is correct.